identity/Cargo.toml

```diff
@@ -31,10 +33,10 @@ harness = false
 default = ["async"]
 
 # Enables async runtime support (Tokio)
-async = ["identity-iota/async"]
+async = ["identity-iota/async", "identity-iota-core/async"]
```

Comment:
`identity-iota-core` should not need an `async` feature.

Reply:
Removed dependency of `identity-iota-core` on `iota-client`, thus removing it's async feature.